tests/components/pvpc_hourly_pricing/test_sensor.py

```diff
@@ -59,27 +61,27 @@ def mock_now():
         )
         assert num_warnings == 1
         assert num_errors == 0
-        assert pvpc_aioclient_mock.call_count == 7
+        assert pvpc_aioclient_mock.call_count == 9
 
         # check that it is silent until it becomes available again
         caplog.clear()
         with caplog.at_level(logging.WARNING):
             # silent mode
             for _ in range(21):
                 await _process_time_step(hass, mock_data, value="unavailable")
-            assert pvpc_aioclient_mock.call_count == 28
+            assert pvpc_aioclient_mock.call_count == 30
             assert len(caplog.messages) == 0
 
             # warning about data access recovered
             await _process_time_step(hass, mock_data, value="unavailable")
-            assert pvpc_aioclient_mock.call_count == 29
+            assert pvpc_aioclient_mock.call_count == 31
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5232061)